tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "outDir": "out",
     "rootDir": "src",
     "sourceMap": true,
-    "strict": true
+    "strict": true,
+    "skipLibCheck": true
```

Comment:
Why do we need `skipLibCheck`?

Reply:
If we do not include that, mocha throws exceptions about the included types:

```
❯ npm run test

> terraform@2.17.0 pretest C:\Users\james\src\hashicorp\vscode-terraform
> npm run compile && npm run lint


> terraform@2.17.0 compile C:\Users\james\src\hashicorp\vscode-terraform
> tsc -p ./

node_modules/@types/jest/index.d.ts:35:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'beforeEach' must be of type 'HookFunction', but here has type 'Lifecycle'.

35 declare var beforeEach: jest.Lifecycle;
               ~~~~~~~~~~

  node_modules/@types/mocha/index.d.ts:2624:13
```

Reply:
I see, thanks for the explanation. We now have conflicting type declarations coming from mocha and jest.